unitTest.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
-USE_GPIO=1 ./build.sh Ninja tests
+export USE_GPIO=1
+./build.sh Ninja tests
```

Comment:
```suggestion
USE_GPIO=1 ./build.sh Ninja tests
```
Keep on the same line. That flag is only useful when building, and using `export` makes that less clear.